assembly/src/tokens/mod.rs

```diff
@@ -170,7 +180,12 @@ impl<'a> Token<'a> {
         assert_eq!(Self::EXPORT, self.parts[0], "not an export");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                ParsingError::missing_param(
+                    self,
+                    "export <module_procedure_name>"
+                )
+            ),
```

Comment:
I'd change this to `export.<procedure_path>`.

Reply:
Changed.